client/plots/violin.interactivity.js

```diff
@@ -99,10 +99,12 @@ export function setInteractivity(self) {
 		//For testing and debugging
 		self.app.tip.d.classed('sjpp-violin-brush-tip', true)
 
-		const [niceStart, niceEnd] =
-			self.config.term.term.type == 'integer' ? [Math.round(start), Math.round(end)] : niceNumLabels([start, end])
+		if (self.displayBrushMenu.called == true) {
```

Comment:
use of this flag is unreliable. you can first brush, then click on label and it breaks again
also it's bad the displayMenu function 3rd arg can be either an object, or number. suggest to change to `displayMenu({event, options, plot})` and `display({event, options, start, stop})`

Reply:
okay. can you check now? I modified the arguments